utilities/boostHistHelpers.py

```diff
@@ -5,7 +5,6 @@
 import logging
 import collections
 from . import common
-from scripts.analysisTools.plotUtils.utility import safeOpenFile, safeGetObject
 import narf
 import ROOT
 
```

Comment:
Small clean up suggestions: can you write ```from utilities import logging, common```  and add 
```logger = logging.child_logger(__name__)``` and rename logging into logger further down in the code. Then remove ```import logging``` and ```from . import common```
also delete imports of narf, ROOT, bh, and collections?